Mathlib/Tactic/Linter/Style.lean

```diff
@@ -514,6 +529,15 @@ def openClassicalLinter : Linter where run stx := do
       Instead, use `open Classical in` for definitions or instances, the `classical` tactic \
       for proofs.\nFor theorem statements, \
       either add missing decidability assumptions or use `open Classical in`."
+    -- Also lint if `Classical.prodDecidable` is added as a local or scoped instance.
+    if let some id := getLocalScopedAttributes? stx then
+      if s!"{id}" == "`Classical.propDecidable" then -- TODO: perform a proper typed comparison
```

Comment:
@adomani Do you know how to make this check typed? The current check is definitely not ready for merging.

Reply:
Never mind, I figured it out.